aikau/src/main/resources/alfresco/core/Core.js

```diff
@@ -399,8 +399,9 @@ define(["dojo/_base/declare",
        * @param {object} payload The payload to publish on the supplied topic
        * @param {boolean} [global] Indicates that the pub/sub scope should not be applied
        * @param {boolean} [parentScope] Indicates that the pub/sub scope inherited from the parent should be applied
+       * @param {String} [customScope] An optional scope to use for this topic-publish
        */
```

Comment:
Worth documenting that customScope requires global and parentScope to both be false?